File: ViewCs/Form1.cs

namespace ViewCs
{

public partial class Form1 : Form
{
    public Form1()
    {
        InitializeComponent();
    }

    private void runCount(System.String message)
    {
    }

    private void SampleControl1_RunButtonClick(object sender, EventArgs e)
    {
        runCount(SampleControl1.InputText);
    }
}

}

File: ViewVb/Form1.vb
Public Class Form1

Private Sub RunCount(ByVal message As String)

End Sub

Private Sub mnuFileExit_Click(sender As Object, e As EventArgs) Handles _
            mnuFileExit.Click
''--------------------------------------------------------------------
''    メニュー「ファイル」－「終了」
''--------------------------------------------------------------------
    Application.Exit()
End Sub

Private Sub mnuRunCount_Click(sender As Object, e As EventArgs) Handles _
            mnuRunCount.Click
''--------------------------------------------------------------------
''    メニュー「実行」－「カウント」
''--------------------------------------------------------------------

End Sub

Private Sub SampleControl1_RunButtonClick(sender As Object, e As EventArgs) _
    Handles SampleControl1.RunButtonClick
''--------------------------------------------------------------------
''    イベントハンドラ
''
''    ユーザーコントロールの「実行ボタンクリック」
''--------------------------------------------------------------------

End Sub

End Class
